src/components/structures/MatrixChat.js

```diff
@@ -188,7 +214,12 @@ module.exports = React.createClass({
         switch (payload.action) {
             case 'logout':
                 if (window.localStorage) {
+                    // preserve our HS & IS URLs for convenience
+                    var hsUrl = this.getCurrentHsUrl();
+                    var isUrl = this.getCurrentIsUrl();
                     window.localStorage.clear();
+                    window.localStorage.setItem("mx_hs_url", hsUrl);
+                    window.localStorage.setItem("mx_is_url", isUrl);
```

Comment:
Tempted to inline this: The variable name isn't giving you any more information than the key name here.